manager/integration/tests/test_basic.py

```diff
@@ -1136,3 +1149,137 @@ def test_attach_without_frontend(clients, volume_name):  # NOQA
 
     client.delete(volume)
     wait_for_volume_delete(client, volume_name)
+
+
+@pytest.mark.coretest
+def test_storage_class_from_backup(volume_name, pvc_name, storage_class, client, core_api, pod_make): # NOQA
+
+    VOLUME_SIZE = str(DEFAULT_VOLUME_SIZE * Gi)
+
+    pv_name = pvc_name
+
+    volume = create_and_check_volume(
+        client,
+        volume_name,
+        size=VOLUME_SIZE
+    )
+
+    wait_for_volume_detached(client, volume_name)
+
+    create_pv_for_volume(client,
+                         core_api,
+                         volume,
+                         pv_name)
+
+    create_pvc_for_volume(client,
+                          core_api,
+                          volume,
+                          pvc_name)
+
+    pvc_spec = create_pvc_spec(pvc_name)
+
+    pod = pod_make() # NOQA
+
+    pod['spec']['volumes'] = [pvc_spec]
+    pod_name = pod['metadata']['name']
+
+    create_and_wait_pod(core_api, pod)
+
+    test_data = generate_random_data(VOLUME_RWTEST_SIZE)
+    write_pod_volume_data(core_api, pod_name, test_data)
+
+    volume_id = client.by_id_volume(volume_name)
+    snapshot = volume_id.snapshotCreate()
+
+    volume_id.snapshotBackup(name=snapshot['name'])
+
+    bv, b = find_backup(client, volume_name, snapshot['name'])
+
+    wait_for_backup_completion(client, volume_name, snapshot['name'])
+    status = wait_for_volume_status(client,
```

Comment:
Seems we don't need to wait for this volume's `LastBackup` to update. Backup is usable once it's done. It may save us some time.

Reply:
fixed.